rites/roman1969/src/particular-calendars/canada.ts

```diff
@@ -196,8 +196,10 @@ export class Canada extends CalendarDef {
     },
 
     // src: mr_fr_2021_ed3
-    dedication_of_consecrated_churches_on_last_sunday_of_october: {
-      drop: true,
+    dedication_of_consecrated_churches: {
+      precedence: Precedences.ProperSolemnity_DedicationOfTheOwnChurch_4b,
+      dateDef: { month: 10, date: 25 },
+      isOptional: true,
```

Comment:
the new format seems to differ with the "last sunday" bits

Reply:
```suggestion
      dateDef: { month: 10, date: 25 },
      isOptional: true,
      alternativeTransferDateDefs: [{ dateDef: { month: 10, lastDayOfWeekInMonth: 0 } }],
```

Reply:
unless i'm misunderstanding?

Reply:
The proper missal for Canada only specifies October 25 and does not suggest transferring it to the last Sunday of October. That’s why I didn’t add an `alternativeTransferDateDefs` here.